### ISSUE TITLE
VHS_VideoCombine fails to import

### ISSUE THREAD

**Issue by Animalgel** [Kosinkadink/ComfyUI-VideoHelperSuite#77]
When launching comfyui portable on windows 11 with everything up to date:

ComfyUI web interface:
"When loading the graph, the following node types were not found:
VHS_VideoCombine
Nodes that have failed to load will show as red on the graph."

Command window:
Import times for custom nodes:
   0.0 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet
   0.0 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-animatediff
   0.0 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-AnimateDiff-Evolved
   0.0 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Frame-Interpolation
   0.0 seconds (IMPORT FAILED): C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite
   0.0 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
   0.1 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Stable-Video-Diffusion
   0.2 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_FizzNodes
   0.2 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Styles
   0.3 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager
   0.7 seconds: C:\Stable_Diffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node

Tried uninstalling and reinstalling from the comfyUI manager... 
Manually deleted the 'ComfyUI-VideoHelperSuite' folder and reinstalled...

![image](https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/assets/152078120/5f17b9fd-55de-48a0-b83a-24c0e18bd437)

**Comment by maestroh** [comment]
I've got the same problem. I have a fresh version of comfy installed with only the reactor and VHS plugins. I believe the problem is the version of opencv installed with reactor. See here for some details and a possible solution. 
https://github.com/Gourieff/comfyui-reactor-node/issues/112

Haven't tried this solution yet so not sure if it works.

**Comment by icequake1969** [comment]
Same problem here.    If you disable the Reactor node or this node, then it runs without error.   Enable both and you get the error.  Fresh Windows and Linux versions of Comfy also give the same exact error.

**Comment by icequake1969** [comment]
Thanks Maestroh, that was the fix.   
 1. Close ComfyUI if it runs
2. Go to the ComfyUI root folder, open CMD there and run:
	- python_embeded\python.exe -m pip uninstall -y opencv-python opencv-contrib-python opencv-python-headless
	- python_embeded\python.exe -m pip install opencv-python==[4.7.0.72](http://4.7.0.72/)

**Comment by manducagiuseppe** [comment]
Here's the solution

  1. Go to the ComfyUI main folder
  2. open the requirements.txt file and add 
        - opencv-python 
        - imageio-ffmpeg
  4. Save requirements.txt
  5. Click on update_comfyui_and_python_dependencies.bat

it worked for me

**Comment by deimoslyric** [comment]
> Here's the solution
> 
> 1. Go to the ComfyUI main folder
> 2. open the requirements.txt file and add
>    
>    * opencv-python
>    * imageio-ffmpeg
> 3. Save requirements.txt
> 4. Click on update_comfyui_and_python_dependencies.bat
> 
> it worked for me

Thanks, this worked for me as well

**Comment by flov** [comment]
I am having the same issue in my runpod. I tried what @icequake1969 wrote
```
pip uninstall -y opencv-python opencv-contrib-python opencv-python-headless
pip install opencv-python==4.7.0.72
```
but it hasn't worked so far

**Comment by flov** [comment]
I stand corrected, the changes worked, together with an installation of ffmpeg with `apt-get install ffmpeg` worked for me

**Comment by HappyGitHubUser** [comment]
> Here's the solution
> 
> 1. Go to the ComfyUI main folder
> 2. open the requirements.txt file and add
>    
>    * opencv-python
>    * imageio-ffmpeg
> 3. Save requirements.txt
> 4. Click on update_comfyui_and_python_dependencies.bat
> 
> it worked for me

where it's update_comfyui_and_python_dependencies.bat ?

**Comment by manducagiuseppe** [comment]
> > Here's the solution
> > 
> > 1. Go to the ComfyUI main folder
> > 2. open the requirements.txt file and add
> >    
> >    * opencv-python
> >    * imageio-ffmpeg
> > 3. Save requirements.txt
> > 4. Click on update_comfyui_and_python_dependencies.bat
> > 
> > it worked for me
> 
> where it's update_comfyui_and_python_dependencies.bat ?

You can find it inside the update directory

**Comment by cangzhen1** [comment]
> > > Here's the solution
> > > 
> > > 1. Go to the ComfyUI main folder
> > > 2. open the requirements.txt file and add
> > >    
> > >    * opencv-python
> > >    * imageio-ffmpeg
> > > 3. Save requirements.txt
> > > 4. Click on update_comfyui_and_python_dependencies.bat
> > > 
> > > it worked for me
> > 
> > 
> > where it's update_comfyui_and_python_dependencies.bat ?
> 
> You can find it inside the update directory
Why did my modification of requirements.txt not take effect?

**Comment by tanobuffone** [comment]
> Here's the solution
> 
> 1. Go to the ComfyUI main folder
> 2. open the requirements.txt file and add
>    
>    * opencv-python
>    * imageio-ffmpeg
> 3. Save requirements.txt
> 4. Click on update_comfyui_and_python_dependencies.bat
> 
> it worked for me

Didn't worked at first because when it updates it rewrittes the requirements.txt , solution:
- First exec the update_comfyui_and_python_dependencies.bat
- Wait until it ask you to press a key to continue with the deps
- Edit now the requirements.txt adding opencv-python and imageio-ffmpeg and save
-  Go back and press any key to continue.

**Comment by Bombyring** [comment]
Go to the main Config UI folder
open it requirements.txt file and add
-opencv-python
-imageio-ffmpeg
Save requirements.txt
Click on update_comfyui_and_python_dependencies.Did the ba
method work? thanks

**Comment by roecks** [comment]
It worked for me. Even though I also uninstalled a ReActor custom node. Not sure whether that, or the additions to requirements.txt and updating ComfyUI worked.

**Comment by ArmadstheDoom** [comment]
> > Here's the solution
> > 
> > 1. Go to the ComfyUI main folder
> > 2. open the requirements.txt file and add
> >    
> >    * opencv-python
> >    * imageio-ffmpeg
> > 3. Save requirements.txt
> > 4. Click on update_comfyui_and_python_dependencies.bat
> > 
> > it worked for me
> 
> Didn't worked at first because when it updates it rewrittes the requirements.txt , solution:
> 
> * First exec the update_comfyui_and_python_dependencies.bat
> * Wait until it ask you to press a key to continue with the deps
> * Edit now the requirements.txt adding opencv-python and imageio-ffmpeg and save
> * Go back and press any key to continue.

Yeah this didn't work either. Still says that the node can't be found. I know it installed them, but it can't find the node. This is a fresh install of both comfy and the plugin, btw.

**Comment by chaoscode** [comment]
I'm having the same issue here. I'm running the latest version of comfyui v1.0.0 on TrueNAS Scale as a docker/custom app. It works fine for everything else besides this node. I think this is broken atm. Can anyone assist? The logs just say it cannot import the node.

**Comment by gabriel20xx** [comment]
I also have this issue, can't find ffmpeg, so node cannot be imported

**Comment by overbound** [comment]
As a last step make sure you reinstall  "VHS Video Combine". I did all the steps above but didn't realize I needed to reinstall within ComfyUI. Derp!

**Comment by hectroggle** [comment]
I fixed it! open "comfyui manager" > "Custom Nodes Manager" > search"VideoHelperSuite" > install > restart > fixed!!

**Comment by 0Eagle** [comment]
from all that has been written, it is completely unclear what to do to add the missing ffmeg.
what exactly should be done to make the windows version of cumfyui see ffmeg?
